### ISSUE TITLE
performance mode - GUI & description rework

### ISSUE THREAD

**Issue by TurboGit** [darktable-org/darktable#6659]
**Describe the issue**
The performance mode has been introduced for slow computer. The naming and the current description could be improved I think.

**Expected behavior**

1. "performance mode" may read as it is better to be activated

I'm proposing to reword it to: **"prefer performance over quality"**

2. the tooltip talk about the thumbnails but it affect also the darkroom. 

Indeed, see my commit about this : https://github.com/darktable-org/darktable/commit/cfc0e5b990faeea41669521a0c4e7f75ec31d3dc

Currently:

"if switched on, thumbnails and previews are rendered at lower quality but 4 times faster"

I propose:

"if switched on, thumbnails and previews are rendered at lower quality but 4 times faster. the darkroom view will be slightly blurred for large zoom (>= 400%)"

**Comment by TurboGit** [comment]
@Nilvus : How does that sound to you ?

@jenshannoschwalm : Would that work for you ? I have double checked that the performance mode now only blurred the display starting with 800% zoom level. Below that there is no difference I can see on my screen. To be conservative I have said >= 400% in the tooltip.

**Comment by parafin** [comment]
Why it's blurred at all? For scales larger than 100% the lowest cairo quality (nearest) should give the correct result. It should be switched to better quality only for scales less than 100%.

**Comment by TurboGit** [comment]
@parafin : You may have a point, but are you saying that NEAREST is faster that GOOD (documentation on this is not clear https://www.cairographics.org/manual/cairo-cairo-pattern-t.html#cairo-filter-t) ? If that's the case then NEAREST is probably the one we want for the performance mode indeed as there is no blurring in this case.

**Comment by parafin** [comment]
I don't know the difference between FAST and NEAREST (they are most likely the same), but they are the fastest and produce correct nonblurred blocky upscaled images.

**Comment by TurboGit** [comment]
So you propose to use FAST only for scale > 100% in performance mode IIUC, right ?

**Comment by TurboGit** [comment]
See #6664 for a proposal based on our discussion.

**Comment by jenshannoschwalm** [comment]
seems good for me too

**Comment by Nilvus** [comment]
> @Nilvus : How does that sound to you ?
> 
> @jenshannoschwalm : Would that work for you ? I have double checked that the performance mode now only blurred the display starting with 800% zoom level. Below that there is no difference I can see on my screen. To be conservative I have said >= 400% in the tooltip.

Naming sound good for me. Much better and clear than actually.

**Comment by maboleth** [comment]
Thumbs up!
